content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.0+"
+maintainer = "Community"
+description = "Scale applications based on Cron Schedules."
+go_file = "cron_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cron` trigger that scales based on a Cron Schedule.
+
+```yaml
+triggers:
+- type: cron
+  metadata:
+    # Required
+    timezone: <time zone the cron should be run>
```

Comment:
Let's use real examples maybe?

Reply:
For example, what do we expect here? What can they provide?

Reply:
sure